go/vt/vtctl/grpcvtctldserver/server_slow_test.go

```diff
@@ -96,7 +96,7 @@ func TestEmergencyReparentShardSlow(t *testing.T) {
 				},
 			},
 			tmc: &testutil.TabletManagerClient{
-				DemoteMasterResults: map[string]struct {
```

Comment:
ugh i'm so sorry / thank you for fixing every one of these!

Reply:
:heart: